apollo-entity/src/test/java/com/spotify/apollo/entity/JsonMatchers.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * -\-\-
+ * Spotify Apollo Entity Middleware
+ * --
+ * Copyright (C) 2013 - 2016 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.apollo.entity;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import okio.ByteString;
+
+import static com.jayway.jsonassert.JsonAssert.with;
+
+class JsonMatchers {
```

Comment:
Yeah I had a brief look at it and stopped considering it as soon as I saw the first example which showcased inline written json in the asserts :)

```
sameJSONAs("{\"friend_ids\":[52, 23, 16]}")
```